packages/core/src/tracing/backend_mappers/mapper.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ */
+
+'use strict';
+
+/**
+ * @type {Record<string, Function|null>}
+ */
+const cachedMappers = {};
+
+/**
+ * Dynamically require the mapper based on span.n.
+ *
+ * @param {string} spanName
+ * @returns {Function|null}
+ */
+function loadMapper(spanName) {
+  if (cachedMappers[spanName]) {
+    return cachedMappers[spanName];
+  }
+
+  try {
+    // Ensures mapper files follow the same naming convention.
+    const mapper = require(`./${spanName}_mapper.js`).transform;
```

Comment:
issue: This whole logic to load a file dynamically can be replaced with **magic getters.** 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get

> const transformFunction = require('./mappers/(span.n));

Reply:
updated the code